.gitignore

```diff
@@ -30,4 +30,5 @@ nuget.exe
 *.*sdf
 *.ipch
 *.sln.ide
-*launchSettings.json
\ No newline at end of file
+*launchSettings.json
+*.Designer.cs
```

Comment:
This would be problematic for regular designer files (which the project has). Could you specify a path for this?